modules/cert-manager-acme-http01.adoc

```diff
@@ -35,13 +35,20 @@ spec:
     solvers:
     - http01:
         ingress:
-          class: openshift-default                                 <4>
+          ingressClassName: openshift-default                                 <4>
 ----
 <1> Provide a name for the cluster issuer.
 <2> Replace `<secret_private_key>` with the name of secret to store the ACME account private key in.
 <3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
 <4> Specify the Ingress class.
 
+.. Optional: If you create the object without specifying `ingressClassName`, use the following command to patch the existing ingress:
+
```

Comment:
```suggestion
+
```
This is not aligned in the preview and needs an update before merge.

Reply:
Ack.